projects/igniteui-angular/src/lib/tree/tree-node/tree-node.component.ts

```diff
@@ -585,6 +585,14 @@ export class IgxTreeNodeComponent<T> extends ToggleAnimationPlayer implements Ig
         this.isFocused = false;
     }
 
+    /**
```

Comment:
I've seen this combination so many times in our code and just want to make it more reusable..

```
/**
     * @hidden @internal
     * Prevent the default action and stop the propagation of the event
     */
    private preventDefaultAndStopPropagation(event) {
        event.preventDefault();
        event.stopPropagation();
    }
```

Just one example

![image](https://github.com/IgniteUI/igniteui-angular/assets/11193764/4f60d88e-414b-4373-955d-3e1d522b1b71)